pkg/utils/checksum.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"io"
+)
+
+type Checksum []byte
+func FileChecksum(buf *bytes.Buffer) (Checksum, error) {
```

Comment:
I noticed this PR decreases the code coverage a bit. Should we add tests for this file?